stored_requests/backends/db_provider/mysql_dbprovider.go

```diff
@@ -75,7 +84,65 @@ func (provider *MySqlDbProvider) ConnString() string {
 		buffer.WriteString(provider.cfg.Database)
 	}
 
-	return buffer.String()
+	// TSL connection
+	if provider.cfg.ServerCa != "" && provider.cfg.ClientCert != "" && provider.cfg.ClientKey != "" {
+		if err := setupTLSConfig(provider); err != nil {
+			return "", err
+		}
+		buffer.WriteString("?tls=prebid-tls")
```

Comment:
Should it be a config value: `prebid-tls`?

Reply:
Thee are 3 new settings in prebid configuration to setup TLS connection- `server_ca`, `client_cert` and `client_key`:
```
stored_requests:
  database:
    connection:
      driver: mysql
      host: 172.18.0.3
      port: 3306
      user: root
      password: PreBidPwd
      dbname: prebid
      server_ca: /home/user/ca.pem
      client_cert: /home/user/client-cert.pem
      client_key: /home/user/client-key.pem
```

Here you can find how to configure MySQL to use encrypted connection:
https://dev.mysql.com/doc/refman/8.0/en/using-encrypted-connections.html

Reply:
Yes, understand. I'm only talking about `tls=prebid-tls`. It's weird to see it hardcoded. We will discuss with the team.

Reply:
The `prebid-tls` string is hardcoded in two places:
- when registering the TLS config
- when referencing it in the connection string
It could be any string.

Reply:
Should we move this to a constant defined at the top of the file?